manifest-parser.js

```diff
@@ -0,0 +1,296 @@
+var ManifestParser = (function() {
```

Comment:
The file moved to static/manifest-parser.js, shouldn't you modify that one instead?